sysutils/ksctl/Portfile

```
@@ -20,19 +20,23 @@ long_description    \
     be efficient and can perform tasks quickly and without the need for \
     additional tools. It is a powerful tool that can be used to perform a \
     wide range of tasks.
-
 categories          sysutils
 installs_libs       no
 license             Apache-2
 maintainers         {gmail.com:herby.gillot @herbygillot} \
                     openmaintainer
+  
+checksums           rmd160  47a433e6b20eccb97d6536e41644658d4e5c324e \
+                    sha256  d1939077c4932b90745b8cea98c5c4885e661957e805884f1a8910bf708acdde \
+                    size    93125
 
-checksums           rmd160  42d2035925871cb7a461d44d3da2a24d50e5b57a \
-                    sha256  1b62fab92fb8051580ebe47d2038d8bf7ace7355ea3f7f02571b5433eb4fc58d \
-                    size    75660
+build.pre_args-append \
+    -ldflags \"-X github.com/ksctl/cli/v2/pkg/config.Version=${version} \
+    -X github.com/ksctl/cli/v2/pkg/telemetry.clientIdentity="macports" \
+    -X github.com/ksctl/cli/v2/pkg/config.KsctlCoreVer="v2.7.6" \
```

Is this version supposed to mismatch? If not, please use `${version}`.

```suggestion
    -X github.com/ksctl/cli/v2/pkg/config.KsctlCoreVer="v${version}" \
```

Also, `${go.package}` might make sense here. That gets implicitly set by `go.setup` above:
```suggestion
    -X ${go.package}/v2/pkg/config.KsctlCoreVer="v${version}" \
```